### ISSUE TITLE
Track level completion

### ISSUE THREAD

**Issue by xen-0sd** [open-manifold/Open-Manifold#30]
Right now, the game is a completely fresh start on each open. There is no indication that any levels have been played or beaten. I think it would be nice to put "Hi-score" and "FC%" (as in % of shapes not missed) values next to the song's genre and BPM information in the level select menu.

**Comment by SalsaGal** [comment]
Where would the game store this data? Probably a bad idea to put it in the level itself so that sending a level doesn't send you someone's score information. Maybe put it in one of the system configuration folders like `%APPDATA%` or `~/.config`?

**Comment by xen-0sd** [comment]
My idea: Have a `scores.json` file in the root folder and read it at the start of the game like how it gets the MOTD.

**Comment by SuperFromND** [comment]
We're both proposing the same thing xen, haha. 

As for how the file itself would be formatted, I'm thinking something like this maybe?
```json
[
    {"level": "level folder name? maybe a checksum of the level.json?", "play_count": 5, "cleared": true, "hiscore": 6969}
]
```

And then display it on level select in the middle like this [new text annotated in yellow]:
![image](https://github.com/open-manifold/Open-Manifold/assets/22881403/e45eb946-5f8a-4584-b055-d1beb4d777f2)

Frankly, the level select screen's metadata text could probably use a rearrangement anyways, it's a bit cluttery. Maybe a new issue is in order to discuss changes to that screen too.

**Comment by SuperFromND** [comment]
![2023-07-15_17-45-37](https://github.com/open-manifold/Open-Manifold/assets/22881403/2cd10456-9478-4447-92c6-780049076491)
High scores are now tracked when a level is completed. They're saved to `hiscores.json`, in the same location that `config.json` is stored. Not *quite* as information-detailed as I proposed, but hopefully good enough for now!

**Comment by SuperFromND** [comment]
I have updated the format and my three proposed stats are now fully implemented as of commit [d9378be](https://github.com/open-manifold/Open-Manifold/commit/d9378be9f5f8b78583817c5808c1a032a08c82d8).
![2023-07-17_10-38-19](https://github.com/open-manifold/Open-Manifold/assets/22881403/21fc84a5-a6da-4437-ac7e-1b9ab1bee184)
